test494940.py

```
@@ -0,0 +1,41 @@
+# Global variable usage and side effects
+global_list = [1, 2, 3]
+
+def add_to_global_list(value):
+    global_list.append(value)  # Modifies global state
+    return global_list
+
+print(add_to_global_list(4))
+print(global_list)  # Global variable state may be unpredictable
+
+# Incorrect usage of mutable default arguments
+def append_to_list(value, my_list=[]):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)